packages/terra-framework-docs/src/terra-dev-site/test/worklist-data-grid/PinnedColumns.test.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import WorklistDataGrid from 'terra-worklist-data-grid';
+
+const gridDataJSON = {
+  cols: [
+    { id: 'Column-0', displayName: 'Patient' },
+    { id: 'Column-1', displayName: 'Location' },
+    { id: 'Column-2', displayName: 'Illness Severity' },
+    { id: 'Column-3', displayName: 'Visit' },
+    { id: 'Column-4', displayName: 'Allergy' },
+    { id: 'Column-5', displayName: 'Primary Contact' },
+    { id: 'Column-6', displayName: 'Generic Order Counts' },
+    { id: 'Column-7', displayName: 'Patient Age' },
+    { id: 'Column-8', displayName: 'Medication History' },
+    { id: 'Column-9', displayName: 'My Relationship' },
+    { id: 'Column-10', displayName: 'Not Selectable', isSelectable: false },
```

Comment:
Do we really need 11 columns?  It seems like we should be able to test pinned columns with like 4 or 5 columns of data.

Reply:
With 4 or 5 columns scrolling will not be present/as apparent on larger screen sizes, the greater number of columns makes it easier to validate the scroll behavior.